app/models/comment.rb

```diff
@@ -26,4 +35,16 @@ def as_indexed_json(_options = {})
       include: user_attributes
     )
   end
+
+  def parents
+    comment_parents = [:question]
```

Comment:
True @andela-oojewale, the question method is defined on the comment to enable easy access to the question from the comment, wherever it was made, since if it was a comment on a question or answer, they all belong to a question  directly or indirectly.